test/powershell/Host/TabCompletion/BugFix.Tests.ps1

```diff
@@ -27,4 +27,12 @@ Describe "Tab completion bug fix" -Tags "CI" {
         $result.CompletionMatches.Count | Should Be 1
         $result.CompletionMatches[0].CompletionText | Should Be "pscustomobject"
     }
+    It "Issue#1345 - 'Import-Module -n<tab>' should work" {
```

Comment:
Currently just following convention already defined in this script.  Created new issue to get this test script cleaned-up: https://github.com/PowerShell/PowerShell/issues/3369